### ISSUE TITLE
validation set objects

### ISSUE THREAD

**Issue by AsherYartsevTech** [AsherYartsevTech/Mask_RCNN#9]
1. what are the images that we didn't touched so far (that we didn't generate leaves, flowers and cucumbers from them)? @orshemesh91 
2.we need to find them, and to produce validation sets and test sets ***as soon as possible*** (this week)
3. if the objects we hold are withdrawn from all of the images from data set --> we must split them again and perform a training from scratch for Sunday meeting

**Comment by simonlousky** [comment]
What is the train / test ratio we want?

**Comment by AsherYartsevTech** [comment]
75% train, 15% validation, %10 test

**Comment by simonlousky** [comment]
On it

**Comment by simonlousky** [comment]
dataset is split in the GT980 compuer, script is ready and generic.
closing.
